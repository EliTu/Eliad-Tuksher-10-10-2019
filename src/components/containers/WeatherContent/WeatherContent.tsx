import React from 'react';
import SearchComponent from '../SearchContainer/SearchContainer';
import WeatherDisplayContainer from '../WeatherDisplayContainer/WeatherDIsplayContainer';

const WeatherContent: React.FC = () => {
	return (
		<>
			<SearchComponent />
			<WeatherDisplayContainer />
		</>
	);
};

export default WeatherContent;
