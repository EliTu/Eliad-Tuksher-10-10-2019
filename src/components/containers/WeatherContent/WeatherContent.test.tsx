import React from 'react';
import { shallow, ShallowWrapper } from 'enzyme';
import WeatherContent from './WeatherContent';
import initialShallowRender from '../../../utilities/test-utilities/initialShallowRender';

describe('WeatherContent component', () => {
	let component: ShallowWrapper;
	beforeEach(() => shallow(<WeatherContent />));

	it('should render without errors', () => {
		initialShallowRender(component);
	});

	it('should render the SearchContainer and WeatherDisplayContainer components', () => {
		expect(component.children().length).toBe(2);
		expect(component.children());
	});
});
