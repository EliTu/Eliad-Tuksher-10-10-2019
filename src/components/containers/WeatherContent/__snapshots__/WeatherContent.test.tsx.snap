// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WeatherContent component should render without errors 1`] = `undefined`;
